package.json

```diff
@@ -46,7 +46,7 @@
     "ember-get-config": "^0.2.2",
     "ember-inflector": "^2.0.0 || ^3.0.0",
     "lodash": "^4.17.11",
-    "pretender": "2.1.1"
+    "pretender": "3.0.1"
```

Comment:
@samselikoff is there a reason why `pretender` is pinned to a specific version? I would like to take advantage of the bug fixes in v3.0.2, but due to the pinning I can't :-/